docs/source/peers/peers.md

```diff
@@ -440,19 +401,19 @@ to be connected to an orderer --- peers can cascade blocks to other peers using
 the **gossip** protocol, who also can process them independently. But let's
 leave that discussion to another time!
 
-Upon receipt of a block, a peer will process each transaction in the sequence in
-which it appears in the block. For every transaction, each peer will verify that
+Upon receipt of a block, a peer will process each transaction in the sequence
+specified in the block. For each transaction, the peer will verify that
 the transaction has been endorsed by the required organizations according to the
-*endorsement policy* of the chaincode which generated the transaction. For
+*endorsement policies* for the chaincode which generated the transaction. For
 example, some transactions may only need to be endorsed by a single
-organization, whereas others may require multiple endorsements before they are
-considered valid. This process of validation verifies that all relevant
+organization, whereas others may require multiple endorsements to be valid. This process of
+validation verifies that all relevant
 organizations have generated the same outcome or result. Also note that this
 validation is different than the endorsement check in phase 1, where it is the
 application that receives the response from endorsing peers and makes the
-decision to send the proposal transactions. In case the application violates
-the endorsement policy by sending wrong transactions, the peer is still able to
-reject the transaction in the validation process of phase 3.
+decision to send the proposal transactions. So even if the application violates
+the endorsement policy by sending incorrect transactions, the peer is still able to
+reject the transaction in the validation process during phase 3.
```

Comment:
You can remove lines 412-416, since it is no longer the application that makes the decision of whether to submit the endorsed transaction. It is the gateway service that does this.

Reply:
ok.